README.md

```diff
@@ -42,10 +42,12 @@ If you wish to try out early Sanitizer implementations, the
 
 ## Explainer
 
-The core API of the Sanitizer is rather simple: There are `Sanitizer` objects,
-and they have a `.sanitize` method that produces a `DocumentFragment` from a
-given text string. Or from another `DocumentFragment`, or even from a whole
-`Document`.
+The core API of the Sanitizer is rather simple: There are `Sanitizer` objects
+which encapsulate the information of what the sanitizer is supposed to do,
+and there is an `Element.setHTML` method which takes a string and (optionally)
+an options dictionary with a `"sanitizer"` key and a `Sanitizer` instance as
+a value. `.setHTML` will parse and sanitize the string it receives, and will
+update the children of the Element it is called on with the result.
 
 Example:
 ```js
```

Comment:
While we're at it, I think we could start explaining/defining two variables for an element and for untrusted input a priori. Repeating the `document.querySelector()` call so often is a bit noisy, isn't it?

Reply:
Done.